README.md

```diff
@@ -19,4 +21,28 @@ HTC-Grid might not be the best choice if :
 
 The full documentation of the HTC grid can be accessed here [https://awslabs.github.io/aws-htc-grid/](https://awslabs.github.io/aws-htc-grid/)
```

Comment:
Is it possible to update the URL of the documentation to : [https://finos.github.io/htc-grid/](https://finos.github.io/htc-grid/) ?